build.gradle

```diff
@@ -1006,6 +1009,37 @@ project(':iceberg-open-api') {
     recommend.set(true)
   }
   check.dependsOn('validateRESTCatalogSpec')
+
+  // Create a custom configuration that extends testFixturesImplementation and is resolvable
+  configurations {
+    testFixturesShadowImplementation {
+      canBeResolved = true
+      canBeConsumed = false
+      extendsFrom testFixturesImplementation
+    }
+  }
+
+  shadowJar {
+    archiveBaseName.set("iceberg-open-api-test-fixtures-runtime")
+    archiveClassifier.set(null)
+    configurations = [project.configurations.testFixturesShadowImplementation]
+    from sourceSets.testFixtures.output
+    zip64 true
+
+    // include the LICENSE and NOTICE files for the runtime Jar
+    from(projectDir) {
+      include 'LICENSE'
```

Comment:
We're actually packaging up more in this runtime than what's identified in the top-level LICENSE/NOTICE files, so we need to make sure that everything is captured.

Reply:
Do we have to do this manually or we use the standard tool for this? 

I don't think we have a standard guidelines for this in "CONTRIBUTING.md". Can we add this?

Reply:
Tried the below plugin with default options, not so useful :( 

https://github.com/jk1/Gradle-License-Report

Reply:
Used https://github.com/jk1/Gradle-License-Report to generate License and manually updated the notice for them.

Please check again.

Reply:
cc: @jbonofre

Reply:
Just spot checking the NOTICE, but it doesn't include the Kite notice from the Iceberg NOICE file.  @bryanck had some way of generating the license/notice files that included transitive dependencies (which is required by ASF).

Reply:
@bryanck: Could you please help me here? I tried the same plugin that you mentioned in your PR here. I used the configuration as  `testFixturesRuntimeClasspath`, because it is a shadow jar for test fixture source sets. 

Also, We need to add the standard way in "contributing.md" for new users to follow it.

Reply:
I used the same plugin, but for both the notices and licenses. Also, I appended to the root license and notice files, the root notice has the Kite notice.

Reply:
@ajantha-bhat I can generate them if you want, I created a custom renderer for the plugin. I'm hoping to contribute that at some point so this is automated.

Reply:
I can also try manually appending the root files contents.

It only generates `licenses`, how did you generate the notice from that plugin?

Reply:
It can generate both, but I added some logic to my custom renderer to dedupe the notices.